vim/ruby_mappings.vim

```diff
@@ -1,7 +1,6 @@
-map <silent> <LocalLeader>rb :wa<CR> :call _RunAll()<CR>
-map <silent> <LocalLeader>rc :wa<CR> :RunRubyFocusedContext<CR>
```

Comment:
I like this binding and think that the loss of it is greater than the (not particularly apparent) value in switching.

If we can reproduce the binding in the new approach, I think there's no downside to the switch.

Reply:
I will work on reproducing this functionality.

To be more clear about benefits of vim-test:
- We immediately gain runners for many more languages (i counted 19)
- We gain access to more strategies. Eventually, I'd like to try using `:terminal` to keep everything within vim itself (both in neovim and now latest vim 8)
- `:TestVisit` Visits the test file from which you last ran your tests (useful when you're trying to make a test pass, and you dive deep into application code and close your test buffer to make more space, and once you've made it pass you want to go back to the test file to write more tests).

And there's even more little stuff if you read through their README.